packages/ui/src/watch/videos/video-player/index.tsx

```
@@ -25,6 +25,47 @@
       nativeAudioTracks: false,
       nativeVideoTracks: false,
     },
+    userActions: {
+      hotkeys: function (event) {
+        const player = this; // Get player instance
```

Biome (1.9.4)

[error] This aliasing of this is unnecessary.

Arrow functions inherits `this` from their enclosing scope.
Safe fix: Use this instead of an alias.


(lint/complexity/noUselessThisAlias)